pkg/taxonomy/data_catalog_schema.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "https://github.com/IBM/the-mesh-for-data/pkg/taxonomy/data_catalog_schema.json",
+    "title": "Data Catalog Taxonomy",
+    "description": "Portion of the taxonomy owned by the data catalog.",
+    "definitions": {
+        "data_sensitivity": {
+            "type": "string",
+            "description": "Level of sensitivity of the data.",
+            "enum": ["confidential", "personal", "special personal"]
+        },
+        "geography_type": {
+            "type": "string",
+            "description": "Geographical resolution",
+            "enum": ["union", "country", "state", "city", "region"]
+        },
+        "geography": {
+            "type": "string",
+            "description": "Name of the geography"
+        },
+        "data_format": {
+            "type": "string",
+            "description": "Format in which the data is stored.",
+            "enum": ["parquet", "csv", "arrow", "json", "table", "avro", "binary"]
```

Comment:
I don't think data_format should be a strict enum...
adding a new data format should not impact the taxonomy

Reply:
If it is not a strict enum, we should remove the enforced validation from the controller code.

Reply:
That's exactly what the taxonomy is for.